scripts/protoc-swagger-gen.sh

```diff
@@ -22,5 +24,4 @@ done
 swagger-combine ./client/docs/config.json -o ./client/docs/swagger-ui/swagger.yaml -f yaml --continueOnConflictingPaths true --includeDefinitions true
 
 # clean swagger files
-rm -rf cosmos
-rm -rf ibc
+rm -rf ./tmp-swagger-gen
```

Comment:
Could you add this to the Makefile's `clean` rule as well please?

Reply:
Updated